### PR TITLE
Add asssets mappings for 1.27.1

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/assets/exchanges_mappings/coinbase.py [hunk 1]
```diff
@@ -13,6 +14,63 @@
     strethaddress_to_identifier('0x221657776846890989a759BA2973e427DfF5C9bB'): 'REP',
     strethaddress_to_identifier('0x362bc847A3a9637d3af6624EeC853618a43ed7D2'): 'PRQ',
     strethaddress_to_identifier('0x0258F474786DdFd37ABCE6df6BBb1Dd5dfC4434a'): 'ORN',
-    strethaddress_to_identifier('0xdeFA4e8a7bcBA345F687a2f1456F5Edd9CE97202'): 'KNC',
+    strethaddress_to_identifier('0xdd974D5C2e2928deA5F71b9825b8b646686BD200'): 'KNC',
```
Comment (LefterisJP) [inline rotkehlchen/assets/exchanges_mappings/coinbase.py:17]: why this change?

File: rotkehlchen/assets/exchanges_mappings/uphold.py [hunk 2]
```diff
@@ -33,7 +33,6 @@
     strethaddress_to_identifier('0xd26114cd6EE289AccF82350c8d8487fedB8A0C07'): 'OMG',
     strethaddress_to_identifier('0x4575f41308EC1483f3d399aa9a2826d74Da13Deb'): 'OXT',
     strethaddress_to_identifier('0x408e41876cCCDC0F92210600ef50372656052a38'): 'REN',
-    strethaddress_to_identifier('0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F'): 'SNX',
```
Comment (LefterisJP) [inline rotkehlchen/assets/exchanges_mappings/uphold.py:36]: why remove this?

**Review by LefterisJP — COMMENTED** [review]
some questions.

File: rotkehlchen/assets/exchanges_mappings/uphold.py (hunk 2 above)
Comment (yabirgb) [reply rotkehlchen/assets/exchanges_mappings/uphold.py:36]: Since all the exchanges needed the same mapping I moved it to common. Common is a mapping that is valid for all the exchanges

File: rotkehlchen/assets/exchanges_mappings/coinbase.py (hunk 1 above)
Comment (LefterisJP) [reply rotkehlchen/assets/exchanges_mappings/coinbase.py:17]: Who is they here?

Comment (yabirgb) [reply rotkehlchen/assets/exchanges_mappings/coinbase.py:17]: coinbase's API

**Review by LefterisJP — APPROVED** [review]
The KNC token fix in another PR
